pycti/connector/opencti_connector_helper.py

```diff
@@ -1130,8 +1161,38 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
             )
             return []
 
-        if entities_types is None:
-            entities_types = []
+        # If directory setup, write the bundle to the target directory
+        if bundle_send_to_directory:
+            self.connector_logger.info(
+                self.connect_name + " sending bundle to directory"
+            )
+            bundle_file = time.strftime("%Y%m%d-%H%M%S-") + str(time.time()) + ".json"
+            write_file = bundle_send_to_directory + "/" + bundle_file + ".tmp"
```

Comment:
I'm not sure that "/" works with all OS, seems to be `os.path.join()` that can be use to be OS independant.

Reply:
Fixed thanks